java-spiffe-core/src/main/java/io/spiffe/svid/jwtsvid/JwtSvid.java

```diff
@@ -290,7 +300,7 @@ private static void validateAudience(final List<String> audClaim, final Set<Stri
 
     private static JwtSignatureAlgorithm parseAlgorithm(JWSAlgorithm algorithm) throws JwtSvidException {
         if (algorithm == null) {
-            throw new JwtSvidException("jwt header 'alg' is required");
+            throw new JwtSvidException("JWT header 'alg' is required");
```

Comment:
it's unrelated, I make that amendment to keep consistency in that file using capitalized 'JWT', and as it doesn't break any functionality I think it's not worth a separate PR to do it. But if you still think it would be better to move to separate PR I can do it.